examples/sync-server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@automerge/example-sync--server",
+  "private": true,
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "start": "node ./index.js"
```

Comment:
Is it worth updating the script in the root `package.json` to point to this new package?

Reply:
Yup good shout